cmd/exo/cmd/vm_create.go

```diff
@@ -137,11 +136,29 @@ var vmCreateCmd = &cobra.Command{
 			return err
 		}
 
-		table := table.NewTable(os.Stdout)
-		table.SetHeader([]string{"Name", "IP", "ID"})
+		sshinfo, err := getSSHInfo(r.ID, ipv6)
+		if err != nil {
+			return err
+		}
+
+		fmt.Printf(`The deployment of %q went well! What to do now?
+
+Connect to the machine
+
+> exo ssh %s --print
+`, r.Name, r.Name)
+
+		printSSHConnectSTR(sshinfo)
+
+		fmt.Printf(`
+Put the SSH configuration into ".ssh/config"
+
+> exo ssh %s --info
+`, r.Name)
+
+		printSSHInfo(sshinfo)
 
-		table.Append([]string{r.Name, r.IP().String(), r.ID})
-		table.Render()
+		println("\nTip of the day: you're the sole owner of the private key. Be cautious with it.")
```

Comment:
println goes to stderr while fmt.Printf goes to stdout. Don't mix unless you have a good reason to mix them.